### DISCUSSION TITLE
Version 3 Migration - Unraid

### DISCUSSION THREAD

**Discussion by Budlyte** [rommapp/romm#740] [Q&A] [answered]
Howdy,
I updated to V3, and having a hard time getting RomM to run now. I keep getting the below error in the RomM logs.

> mariadb.OperationalError: Unknown server host 'mariadb' (-2)

Here's my mariadb config, which is running.
![image](https://github.com/rommapp/romm/assets/4451365/e14d7579-0e70-45e7-ae6c-f77202b70b8d)

RomM config:
![image](https://github.com/rommapp/romm/assets/4451365/92e02134-b3b4-47d5-84f0-3fe24f892ea1)

DB_PASSWD matches MYSQL_PASSWORD in mariadb.

I even tried a fresh build and a config file isn't being generated. It looks like some permissions issues or something maybe?

**Comment by gantoine** [comment]
> Unknown server host 'mariadb'

seems romm can't find a host called `mariadb`. you should verify that romm can communicate with mariadb (you may have to change "romm db host" in your env config)

> I even tried a fresh build and a config file isn't being generated

the config file isn't generated automatically, you can either create one or not use one (its not required)

**Reply by Budlyte** [reply]
I'm not sure what this means
> change "romm db host" in your env config
Change the variable "DB_HOST" to "ROMM_DB_HOST"?

They're bridged to the same network, with mariadb's port 3036 available on the Host network, and the container named "mariadb". I'm not sure what else could be needed in order for romm to communicate with mariadb.
![image](https://github.com/rommapp/romm/assets/4451365/11e0689d-0726-41ee-8821-cf2e58a2a683)

This is a pic of my Docker page after updating RomM. On refresh, RomM is stopped with the same fault above.  (*Edit, I fixed the webui port)

**Reply by gantoine** [reply, marked as answer]
Maybe try using the IP address for mariadb instead, as the host name `DB_HOST`?

**Reply by Budlyte** [reply]
That did it!
I could swear I tried this earlier, but I guess I mistyped it or something.
